### PR TITLE
Genre, Age rating, and English page support for Series

### PR DESCRIPTION
**Pull request by fluzi**
Added support for Genre and Age Rating, as well as English page support for Series.

### CODE REVIEW

File: mir/dlsite/api.py [hunk 1]
```diff
@@ -107,6 +111,34 @@ def _get_series(soup) -> str:
         raise _NoInfoError('no series')
 
 
+_AGE_RATING_PATTERN = re.compile('^Age Ratings|^年齢指定')
+
+
+def _is_age_rating(tag) -> bool:
+    """BeautifulSoup match function for age rating."""
+    return tag.get('class') == ['work_genre'] \
+    and _AGE_RATING_PATTERN.search(tag.parent.parent.th.string)
```
Comment (darkfeline) [inline mir/dlsite/api.py]: Use [PEP8](https://www.python.org/dev/peps/pep-0008/?#id17) style.

```
    return (tag.get('class') == ['work_genre']
            and _AGE_RATING_PATTERN...)

**Comment by darkfeline** [comment]
https://github.com/darkfeline/mir.dlsite/issues/3
https://github.com/darkfeline/mir.dlsite/issues/4

File: mir/dlsite/api.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Modifications copyright (C) 2018 fluzi
```
[the remark this answers is not in the document]
Comment (fluzi) [reply mir/dlsite/api.py]: My apologies, I wasn't sure what I should do for the copyright info.

**Comment by darkfeline** [comment]
Closing since this is too old now

**Pull request closed without merging**